cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -202,4 +202,3 @@ SoundCueCookQualityIndex=-1
 ; This section gets added by default to a blank Unreal project.
 [/Script/WorldPartitionEditor.WorldPartitionEditorSettings]
 CommandletClass=Class'/Script/UnrealEd.WorldPartitionConvertCommandlet'
-
```

Comment:
Is this needed? Did Unreal do this automatically?

Reply:
I'm not sure if this is needed, removing it.